Tests/LiquidProjections.NHibernate.Specs/packages.config

```diff
@@ -1,14 +1,14 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Chill" version="3.0.0" targetFramework="net452" />
-  <package id="FluentAssertions" version="4.19.2" targetFramework="net452" />
+  <package id="Chill" version="3.0.1" targetFramework="net452" />
+  <package id="FluentAssertions" version="5.0.0-beta0001" targetFramework="net452" />

```

Comment:
Should we depend on a beta?

Reply:
It's an official beta which will never go away, so I don't see a problem.